core/consensus/safety/src/lib.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Common utilities for accountable safety in Substrate.
+
+#![forbid(missing_docs, unsafe_code)]
+
+use client;
+use transaction_pool::txpool::{self, PoolApi};
+use node_runtime::{UncheckedExtrinsic, Call};
```

Comment:
I didn't catch this in the first review, but coupling this to the node-runtime is wrong. We'll need general utilities for constructing calls.